contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
+
+I’d like to address a repeated argument against shipping multiple products, let alone all of them.
+
+### You can build more than a product or two
+
+The world of tech is changing fast.
+
+First, there’s just more money at better valuations than when this advice was created. Software companies, us included, have more leverage than ever before - with capital deployed well, we can do more than our predecessors.
+
+Second, software development is far quicker.
+
+Third, we’ve managed this very well so far. 
 
 ## Our strategy
 
-### 1. Provide every tool needed to build successful products
+### 1. Be the source of truth for customer and product data
+
+Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete customer record. This has happened because the entire industry has focused on integration instead of consolidation.
+
+Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
 
-Building a successful product is hard; doing so when you don't understand your customers is even harder.
+We provide customer infrastructure - by providing _every_ customer data related tool in one place, we can:
+
+- Enhance the utility of all the tools when used together
+- Increase trust in data by eliminating complex data stacks
+- Automate everything better than anyone else can, by using AI across this wider data context
+- Continue to provide all the tools companies need as they grow
+
+### 2. Provide every tool needed to build successful products
 
 We aim to offer every tool software teams need to find, sell to, get users, understand who they are, support them and improve their product experience.
 
-### 2. Get in first
+By doing this, we wind up with the perfect customer data record. No integrations needed.
 
-The tools we build help you from your first user to when your product is everything it could be.
+We can then get our AI to work across all of them together, whilst making every individual tool cheaper than the rest of the market - since we provide so many we can charge less.
 
-This means PostHog gives you the best practices, and the industry standard tools, so you can focus on what you do best: building successful products. 
+### 3. Get in first
 
-We offer a generous free tier, no-brainer pricing, and data infrastructure that scales until IPO and beyond.
+Since developers exist first in a startup, by getting in with them early, we are upstream naturally of every single tool they would have otherwise used. Although anyone can pick up our products, this means we can best deliver customer infrastructure to early stage companies so should focus there by default.
```

Comment:
```suggestion
Since developers exist first in a startup, by getting in with them early, we are naturally upstream of every other tool they might have considered using. Although anyone can pick up our products (and lots of mature companies certainly do), this means we can best deliver customer infrastructure to early stage companies, and so should focus there by default.
```